fs/xfstests.py

```diff
@@ -77,6 +77,9 @@ def setUp(self):
                              'xfsdump', 'psmisc', 'sed', 'libacl-devel',
                              'libattr-devel', 'libaio-devel', 'libuuid-devel',
                              'openssl-devel', 'xfsprogs-devel'])
+
+            if self.detected_distro.name == 'rhel' and '8' in self.detected_distro.version:
```

Comment:
Missed suggesting this first, safer to use `version.startswith('8')` so that we can cover all cases.

Reply:
@harish-24 sure make sense will adopt